corehq/ex-submodules/couchforms/models.py

```diff
@@ -194,6 +194,16 @@ def is_deprecated(self):
         return False
 
     @property
+    def is_submission_error_log(self):
+        assert self.doc_type == 'XFormInstance'
+        return False
+
+    @property
+    def is_normal(self):
+        assert self.doc_type == 'XFormInstance'
```

Comment:
Should drop the assert since I assume this is not going to be overridden by the subclasses.